<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestFrame</class>
 <widget class="QFrame" name="TestFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1681</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="finishTestButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Финиш</string>
   </property>
  </widget>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="testComboBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestionButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>След</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startTestButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
